web-services/query/src/main/java/datawave/webservice/query/metric/QueryMetricsBean.java

```diff
@@ -74,6 +75,9 @@ public class QueryMetricsBean {
     @Inject
     private QueryGeometryHandler queryGeometryHandler;
     @Inject
+    @ConfigProperty(name = "dw.basemaps", defaultValue = "{}")
```

Comment:
Do we need to add entries for `dw.basemaps` in `[add|remove]-datawave-configuration.cli` and `default.properties` as we do for others?

Reply:
It's already in there since it gets injected into SimpleQueryGeometryHandler